.gitignore

```diff
@@ -32,3 +32,9 @@ npm-debug.log
 # we ignore priv/static. You may want to comment
 # this depending on your deployment strategy.
 /priv/static/
+
+# Ignore local .terraform cache and .tfstate that shouldn't be committed
+.terraform
+*.tfstate
+*.tfstate.backup
+.terraform.tfstate.lock.info
```

Comment:
We like to insert final newlines in our files. It's in our [`.editorconfig`](https://github.com/newaperio/phoenix_starter/blob/67abc98a572a4b402a1ab2d83ee41969ed2bc8d7/.editorconfig#L5), maybe your editor doesn't respect those settings by default?